manifest.json

```diff
@@ -6,15 +6,15 @@
         "en": "Manage MySQL databases over the web",
         "fr": "Application web de gestion des bases de données MySQL"
     },
-    "version": "5.0.2~ynh1",
+    "version": "5.0.4~ynh1",
     "url": "http://www.phpmyadmin.net",
     "license": "GPL-2.0-only",
     "maintainer": {
         "name": "julien",
         "email": "julien.malik@paraiso.me"
     },
     "requirements": {
-        "yunohost": ">= 3.8.1"
+        "yunohost": ">= 4.0.0"
```

Comment:
Why are we restraining to Buster...? To enforce migrations for security reasons?

Reply:
It's more about encouraging users to migrate to Buster. It will also reduce the number of issues that users face when upgrading applications using PHP7.3 on their server running Strech with PHP7.0 by default.
Feel free to change this if you think it's relevant.